recipes/main/black-pepper-tofu.md

```diff
@@ -0,0 +1,55 @@
+# Black pepper tofu
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-60%25-orange.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-1.5hr-orange.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Average-orange.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**First cooked by:** [@matthewdking](https://github.com/matthewdking/)
+
+**Date:** Week 16 FACN2
+
+**Author of Recipe**: [Yotam Ottolenghi](https://www.ottolenghi.co.uk/black-pepper-tofu-a-shop)
+
+This tasty tofu recipe is a hit with everyone and goes well with rice or noodles. Side dishes are a good idea such as some roasted red peppers or mushrooms.
```

Comment:
I'm not sure you could say it was a hit with everyone, thats just a broad statement.

Reply:
Who was it not a hit with?

Reply:
well me atleast